src/main/resources/org/jenkinsci/plugins/docker/workflow/DockerDSL/help.jelly

```diff
@@ -112,6 +112,13 @@
                 Hexadecimal ID of a running container.
             </p>
         </dd>
+        <dt><code>Container.exec([args, command])</code></dt>
```

Comment:
@hogarthj I think you should add the `args` attribute to `exec` in the same way `Image.run` does.